docs/sources/fundamentals/timeseries-dimensions/index.md

```diff
@@ -101,7 +101,7 @@ For more information, see the documentation on [creating alerts with multiple se
 
 In the case of SQL-like data sources, more than one numeric column can be selected, with or without additional string columns to be used as dimensions. For example, `AVG(Temperature) AS AvgTemp, MAX(Temperature) AS MaxTemp`. This, if combined with multiple dimensions, can result in a lot of series. Selecting multiple values is currently only designed to be used with visualization.
 
-Additional technical information on tabular time series formats and how dimensions are extracted can be found in [the developer documentation on data frames as time series]({{< relref "../../developers/plugins/introduction-to-plugin-development/data-frames#data-frames-as-time-series" >}}).
+Additional technical information on tabular time series formats and how dimensions are extracted can be found in [the developer documentation on data frames as time series](/docs/grafana/latest/developers/plugins/introduction-to-plugin-development/data-frames#data-frames-as-time-series).
```

Comment:
I think this should remain a relative link using the `relref` shortcode because we don't want to take users out of the "next" documentation and into "latest".

Reply:
Note, we found out that this page is mounted, so we now know it needs to be a `docs/reference` link.

Reply:
👍